launch/tier4_perception_launch/launch/object_recognition/detection/camera_lidar_radar_fusion_based_detection.launch.xml

```diff
@@ -392,27 +396,30 @@
   <group if="$(eval &quot;'$(var objects_filter_method)'=='lanelet_filter'&quot;)">
     <include file="$(find-pkg-share detected_object_validation)/launch/object_lanelet_filter.launch.xml" if="$(var use_object_filter)">
       <arg name="input/object" value="objects_before_filter"/>
-      <arg name="output/object" value="near_objects"/>
+      <arg name="output/object" value="$(var output_of_filtered_objects)"/>
       <arg name="filtering_range_param" value="$(var object_recognition_detection_object_lanelet_filter_param_path)"/>
     </include>
   </group>
 
   <group if="$(eval &quot;'$(var objects_filter_method)'=='position_filter'&quot;)">
     <include file="$(find-pkg-share detected_object_validation)/launch/object_position_filter.launch.xml" if="$(var use_object_filter)">
       <arg name="input/object" value="objects_before_filter"/>
-      <arg name="output/object" value="near_objects"/>
+      <arg name="output/object" value="$(var output_of_filtered_objects)"/>
       <arg name="filtering_range_param" value="$(var object_recognition_detection_object_position_filter_param_path)"/>
     </include>
   </group>
 
-  <!-- Merge far_objects and near_objects. This merger will be removed after tracking fusion operates stably. -->
-  <include file="$(find-pkg-share object_merger)/launch/object_association_merger.launch.xml">
-    <arg name="input/object0" value="near_objects"/>
-    <arg name="input/object1" value="radar/far_objects"/>
-    <arg name="output/object" value="$(var output/objects)"/>
-    <arg name="priority_mode" value="0"/>
-    <arg name="data_association_matrix_path" value="$(var object_recognition_detection_object_merger_data_association_matrix_param_path)"/>
-    <arg name="distance_threshold_list_path" value="$(var object_recognition_detection_object_merger_distance_threshold_list_path)"/>
-    <arg name="sync_queue_size" value="$(var far_object_merger_sync_queue_size)"/>
-  </include>
+  <!-- 'radar_long_range_integration' should defined in perception.launch -->
+  <group if="$(eval &quot;'$(var radar_long_range_integration)'=='detection'&quot;)">
+    <!-- Merge far_objects and near_objects. This merger will be removed after tracking fusion operates stably. -->
```

Comment:
Do you want to keep this comment?

Reply:
I want to keep first comment ` <!-- 'radar_long_range_integration' should defined in perception.launch -->` so that user can check `radar_long_range_integration` from this source code.

I think we can remove latter one because this PR make integration stage be switchable.

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/5269/commits/d3d5e3dc0aae6997808b7b079610a3ddbfa7b3c3.